File: backend/app/schemas/market.py
from pydantic import BaseModel


class MarketOverviewResponse(BaseModel):
    country: str
    zone: str
    average_price_eur_mwh: float
    peak_price_eur_mwh: float
    cheapest_hour: str
    market_regime: str
    regime_confidence: float
    risk_status: str
    recommendation: str

File: frontend/types/market.ts
export type MarketOverview = {
  country: string;
  zone: string;
  average_price_eur_mwh: number;
  peak_price_eur_mwh: number;
  cheapest_hour: string;
  market_regime: string;
  regime_confidence: number;
  risk_status: string;
  recommendation: string;
};